### ISSUE TITLE
How should we(ALC Beijing) update the ASF repository metadata?

### ISSUE THREAD

**Issue by wu-sheng** [X-lab2017/open-digger#712]
I want to add some labeled data into OpenDigger to help us for our community analysis.

Label: 

Repos:
-

Orgs/Users:
-

Hi, I noticed there is an ASF definition file in this repo, but I failed in finding the guidance to update it.
I would invite the ALC Beijing team to update this, at least part of this.
Is there any document I can follow?

https://github.com/X-lab2017/open-digger/blob/master/labeled_data/foundations/apache.yml

**Comment by frank-zsy** [comment]
@wu-sheng Thanks for opening the issue.

Users don't need to update the `yml` file themselves. If members under ALC Beijing want to add metadata for the label, they can open an issue like this one and set `Label` as `ACL Beijing` and add `repos` or `orgs/users` name as list.

We will have a GitHub action to parse all the names into id, and we will add the data into `yml` later.

And I assume `ACL Beijing` label should be under `community`?

**Comment by wu-sheng** [comment]
I didn't want to add ALC Beijing label, instead, projects in ASF are belonging to different PMC(s). I want to check whether there is a way to separate them? Or is this recommended?

**Comment by frank-zsy** [comment]
I assume you may want to separate different projects from the ASF foundation? Like `skywalking`, `skywalking-ui`, `skywalking-website` should all belong to `Skywalking`.

If this it is, you can open an issue with `Skywalking` label and all the `repos` belong to it, and we will add a project label as `Skywalking` like [this one](https://github.com/X-lab2017/open-digger/blob/master/labeled_data/companies/alibaba/dubbo.yml).

**Comment by frank-zsy** [comment]
And also I am curious about what kind of repos should be labeled as `Apache`, right now only repos under `apache` org are labeled as `Apache` and other orgs like `dubbo-go` are not labeled as `Apache`. Is this correct or eco projects should also belongs to `Apache`?

**Comment by wu-sheng** [comment]
`dubbo-go` org actually is not a legal org name, especially when it is managed by the Dubbo PMC members. 
Only repositories in the ASF could be seen as a part of official repositories. Such as, at SkyWalking, https://skywalking.apache.org/docs/, SkyAPM org hosts projects of 3rd party, but as a member of our ecosystem.

**Comment by frank-zsy** [comment]
Sorry, actually it's `dubbogo`. So the 3rd party projects belong to certain projects but not to ASF right?

So we can add `Skywalking` project label with all the project and ecosystem orgs and repos as well. But only repos under `apache` org should be labeled as `Apache`, am I right?

**Comment by wu-sheng** [comment]
> Sorry, actually it's dubbogo. So the 3rd party projects belong to certain projects but not to ASF right?

No, they are not.

> So we can add Skywalking project label with all the project and ecosystem orgs and repos as well. But only repos under apache org should be labeled as Apache, am I right?

Yes. And this could be done automatically. In the ASF, all TLPs have their unique repo prefix, such as all `skywalking-*` repos definitely belong to the SkyWalking project.

**Comment by frank-zsy** [comment]
Understood, right now in the issue all repos should be listed with full name and with this situation we can build a process to do this for ASF and we can do this under `Project` label type.

Could PMCs of ASF start issues for their projects address the label and prefix of the repos and we will add the data after.

Just like:

```
Label: SkyWalking

Repos:
- apache/skywalking-*

Orgs/Users:
- SkyAPM
```

Is this feasible for you?

**Comment by wu-sheng** [comment]
```
Orgs/Users:
- SkyAPM
```

What does this mean? Ecosystem? This org just hosts some of the ecosystem projects

**Comment by frank-zsy** [comment]
This means the repos under this org will be labeled as `SkyWalking` but not `Apache`, this is the decision you should make whether the ecosystem repos should belong to this label. And just remove this line if not.

**Comment by wu-sheng** [comment]
> This means the repos under this org will be labeled as `SkyWalking` but not `Apache`, this is the decision you should make whether the ecosystem repos should belong to this label. And just remove this line if not.

Yes, please. We even list them on the ASF website, they should be tagged as skywalking but not ASF.

**Comment by frank-zsy** [comment]
Understood, I will open an issue for `Skywalking` and be an example for other projects, and I will open a PR about this and invite you as reviewer~.

**Comment by frank-zsy** [comment]
closed by #715
